src/Fable.Analyzers/StdOutAnalyzer.fs

```diff
@@ -0,0 +1,63 @@
+﻿module Fable.Analyzers.StdOutAnalyzer
+
+open System
+open FSharp.Analyzers.SDK.TASTCollecting
+open FSharp.Compiler.Symbols
+open FSharp.Compiler.Text
+open FSharp.Analyzers.SDK
+
+let namesToAvoid =
+    set
+        [
+            "System.Console.Write"
+            "System.Console.WriteLine"
+            "System.Console.WriteAsync"
+            "System.Console.WriteLineAsync"
+            "Microsoft.FSharp.Core.Operators.print"
```

Comment:
@nojaf Shouldn't the last 3 be these instead?
```fs
    "Microsoft.FSharp.Core.Printf.PrintFormat"
    "Microsoft.FSharp.Core.Printf.PrintFormatToError"
    "Microsoft.FSharp.Core.Printf.PrintFormatLine"
    "Microsoft.FSharp.Core.Printf.PrintFormatLineToError"
```

Reply:
Thanks, those names are wrong indeed: https://github.com/dotnet/fsharp/blob/main/src/FSharp.Core/fslib-extra-pervasives.fs#L267-267

I'll check them.